packages/renderer/vite.tests.setup.js

```diff
@@ -22,7 +22,16 @@ import 'vitest-canvas-mock';
 import typescript from 'typescript';
 import { expect } from 'vitest';
 
-global.window.matchMedia = () => {};
+global.window.matchMedia = query => ({
```

Comment:
issue: this file should not be modified

(no difference for this file between upstream and downstream)

https://github.com/podman-desktop/podman-desktop/blob/main/packages/renderer/vite.tests.setup.js

if there is an enhancement, need to go first to upstream